<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>346</width>
     <height>462</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pnConnect">
        <property name="text">
         <string>Connet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pnDisconnect">
        <property name="text">
         <string>Disconnet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="leHost">
        <property name="text">
         <string>naver.com</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lePort">
      <property name="text">
       <string>80</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteSend"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="pteMessage">
        <property name="plainText">
         <string>GET / HTTP/1.1
Host: naver.com

</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSsl">
        <property name="text">
         <string>SSL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
